docs/rules/one-var.md

```diff
@@ -46,6 +63,15 @@ The following patterns are considered warnings:
 function foo() {
```

Comment:
"The following patterns are considered warnings:" above this is duplicated, and the second can be removed.